### PR TITLE
some modifications on code readability

### PR DESCRIPTION
**Pull request by mittwerk**

### CODE REVIEW

File: app/src/main/java/com/jerboa/api/Http.kt [hunk 1]
```diff
@@ -121,7 +121,7 @@ object API {
         }
     }
 
-    suspend fun createTempInstance(
+    private suspend fun createTempInstance(
```
Comment (MV-GH) [inline app/src/main/java/com/jerboa/api/Http.kt:124]: Needs to stay public, not used rn publicly but can in the future, no gain by restricting it

**Comment by MV-GH** [comment]
Also run `./gradlew formatKotlin`

File: app/src/main/java/com/jerboa/api/Http.kt (hunk 1 above)
Comment (mittwerk) [reply app/src/main/java/com/jerboa/api/Http.kt]: Understand

**Comment by MV-GH** [comment]
Pipeline got stuck for some reason, try pushing another commit

**Comment by mittwerk** [comment]
this should definitely work

File: app/src/main/java/com/jerboa/Utils.kt [hunk 2]
```diff
@@ -459,10 +465,15 @@ suspend fun openLink(
     val communityUrl = looksLikeCommunityUrl(parsedUrl)
 
     if (userUrl != null && (formatted || API.checkIfLemmyInstance(url))) {
-        val route = Route.ProfileFromUrlArgs.makeRoute(instance = userUrl.first, name = userUrl.second)
+        val route =
```
Comment (MV-GH) [inline app/src/main/java/com/jerboa/Utils.kt:468]: How is this more readable?

Comment (mittwerk) [reply app/src/main/java/com/jerboa/Utils.kt:468]: probably this is moving the line down for those who don't have widescreen monitors

Comment (MV-GH) [reply app/src/main/java/com/jerboa/Utils.kt:468]: Was this done through a Linter? ktlint? ifso take over the extra rules have added in my other pr (expand search).

**Review by dessalines — CHANGES_REQUESTED** [review]
Please revert the formatting related PRs, and just leave in your smaller commits, to make this easier to read. Then run `./gradlew formatKotlin`

**Pull request closed without merging**